### PR TITLE
first commit: done

### PR DESCRIPTION
**Pull request by kabilza**
Added questions, answers and link related to those question for further study.

### CODE REVIEW

File: django-match-left-and-right/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Matching Django-related question and answers.
+### Question
+Match the letters to the correct sentence or question.
+
+```sh
```
Comment (jbrucker) [inline django-match-left-and-right/README.md]: The triple-backtick is for iterally formatted stuff,
not text.  Don't use it here.

File: django-match-left-and-right/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+1. This file is for redirecting webpages and rendering webpages from request. (Could be a normal function or a class based)
+__
+2. User needs to put their desire address of the website using 'path' function in this bracket([]) named:
```
Comment (jbrucker) [inline django-match-left-and-right/README.md]: their -> his

File: django-match-left-and-right/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+__
+3. To put web html templates, user needs to first create a folder named template, then create folder named "something" inside the template folder to make django reads html file properly.
+__
+4. This built-in django accepts (at least) these 3 arguments: request, "/*.html/", context. What is the name of it and how to import it from?
```
Comment (jbrucker) [inline django-match-left-and-right/README.md]: grammar "built-in django" makes no sense.  Missing word?

File: django-match-left-and-right/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+__
+5. When testing the database, user should use python3 manage.py shell, then type WHAT?
```
Comment (jbrucker) [inline django-match-left-and-right/README.md]: "To test the database interactively"

File: django-match-left-and-right/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+__
+6. With your html templates, to use the picture within the folder, you need {% "something here" %} to be able to recall the file within that location.
```
Comment (jbrucker) [inline django-match-left-and-right/README.md]: Wording needs improvement.  Don't understand what you're trying to write.
Be precise in wording "use the picture within the folder",  what is "use" what is "the folder"?
"that location" is what location?
Must fix this before can approve for merge.

File: django-match-left-and-right/README.md (hunk 1 above)
Comment (kabilza) [reply django-match-left-and-right/README.md]: Fixed

File: django-match-left-and-right/README.md (hunk 2 above)
Comment (kabilza) [reply django-match-left-and-right/README.md]: Fixed

File: django-match-left-and-right/README.md (hunk 3 above)
Comment (kabilza) [reply django-match-left-and-right/README.md]: Fixed: changed to "built-in django function"

File: django-match-left-and-right/README.md (hunk 4 above)
Comment (kabilza) [reply django-match-left-and-right/README.md]: Fixed

File: django-match-left-and-right/README.md (hunk 5 above)
Comment (kabilza) [reply django-match-left-and-right/README.md]: Fixed: "With your html templates, to use the picture and CSS style inside the "static/image" or "static/css" folder, you need to add this line "{% "something here" %}" at the top of the html file for django to be able to recall the image and css file from the static folder."

**Comment by jbrucker** [comment]
I'll give you credit for creating a problem, but not merging it into master.

These questions do not involve important software process knowledge
or development skill/practices that everyone should know.  It just relates
to Django.

**Pull request closed without merging**